### PR TITLE
Add script to wrap skoot tool

### PR DESCRIPTION
**Pull request by ajssmith**

### CODE REVIEW

File: arn-topology/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+To complete this tutorial, do the following:
+
+* [Prerequisites](#prerequisites)
+* [Step 1: Set up the demo](#step-1-set-up-the-demo)
+* [Step 2: Define Topology Values](#step-2-define-topology-values)
+* [Step 3: Generate Deployment Files](#step-3-generate-deployment-files)
+* [Step 4: Deploy Application Router Network](#step-4-deploy-application-router-network)
+* [Next steps](#next-steps)
+
+## Perquisites
```
Comment (ChugR) [inline arn-topology/README.md]: typo - Prerequisites

File: arn-topology/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+You must have access to the OpenShift clusters that you are deploying
```
Comment (ChugR) [inline arn-topology/README.md]: clarity - You must have access to the OpenShift clusters to which you are deploying the application routing network.

File: arn-topology/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+   $ export SKUPPER_PUBLIC_CLUSTER_COUNT=2
+   $ export SKUPPER_PRIVATE_CLUSTER_COUNT=1
+   $ export SKUPPER_NAMESPACE="arn-demo"
+   $ export SKUPPER_PUBLIC_CLUSTER_SUFFIX_1=""mycluster1.devcluster.openshift.com"
+   $ export SKUPPER_PUBLIC_CLUSTER_SUFFIX_2=""mycluster2.devcluster.openshift.com"
+   $ export SKUPPER_PUBLIC_CLUSTER_NAME_1="us-east"
+   $ export SKUPPER_PUBLIC_CLUSTER_NAME_2="us-west"
+   $ export SKUPPER_PRIVATE_CLUSTER_NAME_1="on-prem"
+   ```
+
```
Comment (ChugR) [inline arn-topology/README.md]: Add note about public and private cluster count upper bounds.

**Comment by ajssmith** [comment]
Thanks for the review comments. I addressed typos and expanded support to 3x3 clusters. After that, they should probably use the network creator.

**Pull request merged**